docs/content/any/project/changelogs/NEXT.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Release 2021-05-DD
+menuTitle: 2021-05-DD
+any: true
+description: >-
+  Changelog for Release 2021-05-DD (sqlalchemy-oso 0.6.2) containing new features,
+  bug fixes, and more.
+draft: true
+---
+
+## `sqlalchemy-oso` 0.6.2
+
+### Other bugs & improvements
+
+- Authorized sessions now disable [baked queries][] by default because they can
+  lead to authorization backdoors. If you understand the risks and still want
```

Comment:
I would consider phrasing this differently (not using the word "backdoor").  Maybe authorization "bypass", or "incorrectly cached authorization queries" (more specifically describes what is happening).